src/main/java/frc/robot/commands/ArmControl.java

```diff
@@ -0,0 +1,48 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.command.Command;
+import frc.robot.lib.LogitechController;
+import frc.robot.Robot;
+
+public class ArmControl extends Command {
+  public ArmControl() {
+    // Use requires() here to declare subsystem dependencies
+    // eg. requires(chassis);
```

Comment:
You'll likely need to requires() the CargoIntake subsystem

Reply:
wouldn't that stop any of the roller commands from running while manualarmcontrol was running?

Reply:
Ahh that's right please disregard my original comment for now. Really good catch Rahul! This definitely could cause issues and frustrations down the road if we don't think this fully through.

In the ideal case, we want to control them serially but I can see how the arm will need to call commands to maintain the position while we try to score a cargo.